src/components/Papers/CategoryItemByPaper.jsx

```diff
@@ -18,33 +19,49 @@ const CategoryItemByPaper = ({ papers, category, isLast, onClick }) => {
   const scannerT = useScannerI18n()
   const isStacked = papers.length > 1
 
+  const countPapersAboutToExpire = papers.filter(
+    paper =>
+      paper.metadata.expirationDate &&
+      paper.metadata.noticePeriod !== 0 &&
+      !paper.metadata.hideExpirationAlert
+  ).length
```

Comment:
It will count :
- paper already expired
- paper that will expire in 1 year or 10 years

No ?

Reply:
le 2 n'est pas gênant en soit ça dépend de la noticePeriod qu'on a mis. Mais oui il faut modifier la logique 👍